index.html

```diff
@@ -9,7 +9,9 @@
   </head>
   <body>
     <div class="book-container">
-      <img src="images/book-cover.png" class="book-cover" alt="book-cover">
+      <div class="cover-container badge">
```

Comment:
Why did you use two classes? are you panning using them separately? I think you can merge them into one.